### ISSUE TITLE
WPF dotnet Core 3.1 - Could not load DYMO.CrossPlatform.Common

### ISSUE THREAD

**Issue by avazon** [dymosoftware/DCD-SDK-Sample#16]
I would like to use the SDK from dontet core 3.1 WPF application. I downloaded the sample for dotnet 4.x and it works. But when I create a dotnet core 3.1 project I get the error message:

Could not load file or assembly 'DYMO.CrossPlatform.Common, Version=1.3.1.160, Culture=neutral, PublicKeyToken=null'. File not found.

The file exists and information of file is 1.3.1.160

**Comment by dymosoftware** [comment]
Hi @avazon  & @vmark99, we are working on it. Once a beta version is ready we'll let you know.

**Comment by Jhultink** [comment]
Also looking to use this with .NET Core 3.1

**Comment by AndyHale** [comment]
Would be great for .NET 5 too :)

**Comment by mtbayley** [comment]
Please! .Net5!

**Comment by theOzzieRat** [comment]
Same issues here. Trying to use .NET 5.
@dymosoftware any news on an update? It's been almost 4 months since you said you where working on it!!

**Comment by gknierim** [comment]
Yes - please update for .NET 5.0.  Trying to write a Blazor app and this would be much needed.  As of now, trying to use the JS framework but it doesn't work with the JSInterop.  Has anyone got the dymo.connect.framework.js to work from a web in Blazor?

**Comment by DaveSlinn** [comment]
+1 For .NET 5 support.

**Comment by theOzzieRat** [comment]
@dymosoftware an update would be nice.

**Comment by DaveSlinn** [comment]
@dymosoftware - this is a real blocker for us.  The majority of our code is .NET 5. The only code still running on 4.7 is the stuff to send labels to the printers.  Any thought to open sourcing the code?  Perhaps the community could assist with migration.  After all, the software isn't a commercial product - the printer and the labels are.

**Comment by DaveSlinn** [comment]
With the pending release of Visual Studio 2022 - only .NET 6 will be supported - which means .Net Framework 4.8 or earlier will not be supported at all, which would prevent us from moving to it.  It's been 8 months since the comment above stating they were working on it and a beta would be available.  How can we move this along?  Let us help @dymosoftware!

**Comment by AndyHale** [comment]
Also happy to help with this @dymosoftware.  We're in a similar position where Dymo prints will shortly be our last bit of .Net Framework code.  Happy to contribute.

**Comment by PerryCodes** [comment]
It's not promising when a company doesn't respond for over a year...

**Comment by Pelice** [comment]
news for a dotnet core 5/6 sdk?

**Comment by bertenvdb** [comment]
Also waiting for a .NET 6 compatible SDK...

**Comment by eljayg** [comment]
From #51:
> DymoSDK nuget is currently targeting netstandard2.0 as of today. It will provide multi targeting for net5, net6 in future release. We don't have timeline for the project yet.

Anyone care to share what they're doing in the mean time until there's a fix? Running on .NET 4.7 or 4.8 until then?

**Comment by Makin-Things** [comment]
@dymosoftware 
I have given up on Dymo and replaced all my label printers with an alternate brand. I will never purchase another product from Dymo or the Rubbermaid group since the level of support is non-existent. They promised a beta well over a year and a half ago to target a newer .net, that is now also unsupported software. So they need to start again (if they had actually started in the first place).

**Comment by Pelice** [comment]
> Anyone care to share what they're doing in the mean time until there's a fix? Running on .NET 4.7 or 4.8 until then?

I build a dedicated small app just for printing in .net 4.8 . I haven't found other ways..

**Comment by JLOrdaz** [comment]
I have the same issue. Not working.  The error is: System.IO.FileNotFoundException: Could not load file or assembly 'DYMO.CrossPlatform.Common, Version=1.4.3.37, Culture=neutral, PublicKeyToken=null'.

**Comment by cmwales** [comment]
Its been three years. I doubt they are going to fix it.

**Comment by gkawin** [comment]
I gave up on creating a Printer Web API for the internal network.

**Comment by asifiqbalpatel** [comment]
Is there any update on this bug, I am still facing it
